core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -801,67 +820,76 @@ public void run()
       System.exit(0);
     }
 
-    try
-    {
-      workload.init(props);
-    }
-    catch (WorkloadException e)
-    {
-      e.printStackTrace();
-      e.printStackTrace(System.out);
-      System.exit(0);
-    }
+    final Tracer tracer = new Tracer.Builder("YCSB " + workload.getClass().getSimpleName()).conf(conf).build();
+
+    try (final TraceScope span = tracer.newScope("Client#workload_init")) {
+
+      try
+      {
+        workload.init(props);
+      }
+      catch (WorkloadException e)
+      {
+        e.printStackTrace();
+        e.printStackTrace(System.out);
```

Comment:
Thanks for the clarification guys, nice discussions on HTrace. 

@busbey - Code LGTM as is then.